ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -238,13 +243,63 @@ export default class AdvancedTab extends PureComponent {
     );
   }
 
+  renderToggleStxOptIn() {
+    const { t } = this.context;
+    const { stxOptIn, setStxOptIn } = this.props;
+
+    const learMoreLink = (
+      <ButtonLink
+        size={ButtonLinkSize.Inherit}
+        textProps={{
+          variant: TextVariant.bodyMd,
+          alignItems: AlignItems.flexStart,
+        }}
+        as="a"
+        href="https://support.metamask.io/hc/en-us/articles/9184393821211"
```

Comment:
Either this URL will be updated or we change content there.

Reply:
We have a bunch of URLs in this file:  `ui/helpers/constants/zendesk-url.js`

Reply:
Good to know. This one is on metamask.io, so we would have to rename that file from zendesk-url.js to something like urls.ts. Another option is to have this in shared/constants/smartTransactions.ts, so it would be together with other STX related constants. What do you think?